publish-package.sh

```diff
@@ -5,20 +5,20 @@ set -e
 CURRENT_BRANCH=`git name-rev --name-only HEAD`
 
 if [ $CURRENT_BRANCH != 'master' ] ; then
```

Comment:
Right, if we'd used `TRAVIS_BRANCH` we would have a problem. While looking into the travis environment vars, we might be better of using `TRAVIS_TAG`, because then we don't need the `git tag ..` trickery